invoices/invoiceregistry_test.go

```diff
@@ -1,4 +1,4 @@
-package invoices
+package invoices_test
```

Comment:
I think that underscores are against package naming conventions (although this change is only temporary).

Reply:
It's a common go pattern.

Let's say I have a package called `invoices` in a folder with two files:  `invoices.go` and `invoices_test.go`. If I start  `invoices_test.go` with `package invoices` I will be able to call any non exported function from the invoices package.
Sometimes, people want to make more "realistic" tests that relay only on the exported features. For that reason Go allows you to define files in the same folder with a different package name. If in your business logic lives in the files `package mypackage` you can put other files in the same folder for `package mypackage_test`.

Go won't complain about having two different packages in the same folder. The tooling will work properly (go test) and everything in `mypackage_test` won't be able to access to non exported vars, consts,  functions etc from `mypackage`